web/src/test/auto/headless/engine/main/headless/languageProcessor.spec.js

```diff
@@ -43,8 +43,8 @@ describe('LanguageProcessor', function() {
     it('has expected default values after initialization', function () {
       let languageProcessor = new LanguageProcessor(worker, new TranscriptionCache());
 
-      // These checks are lifted from the keyboard-processor init checks found in
-      // common/web/keyboard-processor/tests/cases/basic-init.js.
+      // These checks are lifted from the keyboard init checks found in
+      // engine/keyboard/tests/cases/basic-init.js.
```

Comment:
```suggestion
      // engine/js-processor/tests/cases/basic-init.js.
```

Assuming changes are made to the test location per my other comments.